distribution/std-lib/Table/src/Data/Order_Rule.enso

```
@@ -0,0 +1,24 @@
+from Base import all
+
+type Order_Rule
+    ## A rule used for sorting table-like structures.
+
+       Arguments:
+         - column: a value representing the underlying storage this rule is
+                   sorting by. This type does not specify the underlying
+                   representation of a column, assuming that the sorting engine
+                   defines its own column representation.
+         - comparator: a function taking two elements of the underlying column
+                       and returning an `Ordering`. The function may be
+                       `Nothing`, in which case a natural ordering will be used.
+                       Note that certain table backends (such us database
+                       connectors) may choose to ignore this field.
```

Maybe instead `choose to ignore` it should say `not support this field being set to anything other than Nothing`?

What I'm trying to convey is that the backend should check this field and fail with an error if it cannot support the operation, instead of silently ignoring its value (which could be super-confusing).